lib/burnside/intrinsics.cc

```diff
@@ -72,8 +72,17 @@ class MathRuntimeLibrary {
   Map library;
 };
 
-/// enum used to templatized code gen for intrinsics that are alike.
+/// enum used to templatize code gen for intrinsics that are alike.
 enum class Extremum { Min, Max };
+// There are many ways to deal with NaNs in Min and Max.
+// Existing behaviors are listed below.
```

Comment:
Perhaps you should describe the behaviors in this comment.

Reply:
Done in PR update.